bin/hledger-prices.hs

```diff
@@ -19,9 +20,10 @@ import Hledger.Cli
 ------------------------------------------------------------------------------
 cmdmode = hledgerCommandMode
   [here| prices
-Print all prices from the journal.
+Print all market prices from the journal.
   |]
-  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings instead of market prices"]
+  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings"
+  ,flagNone ["inverted-costs"] (setboolopt "inverted-costs") "print transaction inverted prices from postings also"]
   [generalflagsgroup1]
```

Comment:
I used to think about transaction as a temporary account to which we deposit and from which we withdraw money.
I guess I owe some use cases here to make it clear.

```
2017/1/1
    assets:usd  $-107
    assets:eur  100 EUR

2017/1/2
    assets:usd  $-107  @@ 100 EUR
    assets:eur  100 EUR

2017/1/3
    assets:usd  $-107
    assets:eur  100 EUR @ $1.07
```

All transactions represent a valid record of currency exchange. But first two and the last one bear a different price information. 

```
P 2017-01-01 $ 0.93457943925233644859813084112149532710280373831775700934579439252 EUR
P 2017-01-02 $ 0.935 EUR
P 2017-01-03 EUR $1.07
```

You can argue that last version is more preferred because you buy EUR for USD.
But consider that I'm abroad and pay with my credit card:

```
2017/1/1 Auchan supermarket
    assets:usd  $-107
    expenses  70 EUR
    expenses  30 EUR

2017/1/2 Auchan supermarket
    expenses  70 EUR
    expenses  30 EUR
    assets:usd  $-107
```
If you'll look at bank receipt you probably will see only one amount/rate. I.e. second transaction that will have two postings with prices will match operation more distantly than the first one.